### ISSUE TITLE
Attach the video explainers from the 2026-09-16 redo run

### ISSUE THREAD

**Issue by opendroid** [opendroid/the-infinity#501]
Refs #384, #386, #403, #437, #439, #495, #499, ADR-0017.

## The run

`npm run find:videos -- search --concepts --redo-empty --redo pretraining,tokenization,generative-adversarial-network`

```
482 concept target(s); 482 already done, 71 to go
  (--redo-empty: retrying 68 that found nothing) (--redo: 3 named target(s) re-queried)
7181 units spent. 1227 candidate(s) across 482 target(s)
```

This is the **thin end of the corpus** — every concept that two prior passes left without a
video. 71 searches produced candidates for 14 targets, of which **3 are worth attaching**.

## What this issue covers

| | |
|---|---|
| **Attach** | 3 entries, verified against real YouTube |
| **Dismiss** | 44 targets — 41 new, 3 replacing a stale reason |
| Corpus after | 911 explainers, 120 dismissals |

## Two things went wrong, both worth recording

**1. The run used pre-#499 code.** The header says `retrying 68`, not 44. Running the merged
`pending()` against that same checkpoint returns **44**, and `constrained-decoding`,
`sandboxing`, `curriculum`, `guardrail` and `refusal` — all of which have videos — were
searched anyway. The clone was four commits behind, so #499's fix wasn't in it and the 2,400
units it exists to save were spent. The fix is correct; it simply wasn't there.

**2. `--redo` on an unchanged query is a no-op, and I advised it anyway.** I claimed
`pretraining`, `tokenization` and `generative-adversarial-network` were "a query problem, not
a YouTube problem" and worth a `--redo`. But `--redo` re-runs the **same query**:

| target | before | after |
|---|---|---|
| `pretraining` | filler | **0 candidates** |
| `tokenization` | 2 candidates | **identical 2 candidates** |
| `generative-adversarial-network` | 1 walkthrough | **0 candidates** |

300 units for nothing. `--redo` is for after a scorer or query change (#403 says so); I used
it as a re-roll. Those three dismissal reasons are rewritten to say exactly that, which is
why 3 of the 44 replace rather than add.

## The picks, and rank again being no guide

| node | score | |
|---|---|---|
| `determinism-and-reproducibility` | **0.39** | Aurélien Géron — *Deep Learning Reproducibility with TensorFlow*. The author of *Hands-On ML*, on precisely the node's subject: same seed, different model |
| `logical-constraint` | **0.50** | John Levine — *Constraint Satisfaction: introduction*, 122k views. A real CS lecture; CSP is the formalism the node describes |
| `weight-averaging` | **0.48** | UAI 2018 — *Averaging Weights Leads To Wider Optima*. **The SWA paper's own conference talk**, and the lowest-scored of that target's three candidates |

## The 44 dismissals, in three kinds

- **11 read and rejected.** `proprioception` is the one to enjoy: the two popular candidates
  are *human* proprioception, from a chiropractor and a fitness channel. `energy-cost-of-training`
  returned an XGBoost time-series tutorial.
- **30 searched twice, zero candidates both times.** Not "nothing good" — nothing at all.
  Recorded as such so `--redo-empty` stops paying for them, and `--redo <id>` after a query
  change remains the way back.
- **3 rewritten**, per the `--redo` failure above.

## A mistake caught before it shipped

My first dismissal file had **52** empty targets, 22 of which already have a video. Dismissing
a covered concept contradicts its own explainer, and `staleDismissals` would have failed the
build. Dropped before the file left the sandbox, and the generator now asserts that no
dismissal names a covered concept.

## Acceptance criteria

- [ ] 3 entries attached; `validate:content` and `check:explainers --offline` clean at 911
- [ ] 44 dismissals applied — 41 new, 3 replacing — and the **76 untouched reasons
      byte-identical**, asserted rather than eyeballed
- [ ] `derived.golden.json` regenerated and unchanged
- [ ] No dismissal names a concept that has a video (`staleDismissals` passes)
